DotNetHowToSecurityTrimming/DotNetHowToSecurityTrimming/SecuredFiles.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Microsoft.Spatial;
+using Newtonsoft.Json;
+
+// The SerializePropertyNamesAsCamelCase attribute is defined in the Azure Search .NET SDK.
+// It ensures that Pascal-case property names in the model class are mapped to camel-case
+// field names in the index.
+[SerializePropertyNamesAsCamelCase]
+public partial class SecuredFiles
+{
+    [System.ComponentModel.DataAnnotations.Key]
+    [IsFilterable]
+    public string FileId { get; set; }
+
+    [IsSearchable, IsFilterable, IsSortable]
```

Comment:
Small nit.  Can we keep the properties in a consistent order?  For example, the next field is [IsFilterable, IsSortable, IsFacetable]